components/gitpod-protocol/src/webhook-event.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+export interface WebhookEvent {
+    id: string;
+    creationTime: string;
+    type: "push" | string;
+
+    authorizedUserId?: string;
```

Comment:
A comment would be golden. :sparkles: 
I still recall from our discussion earlier today, but tomorrow-me will have forgotten for sure :grimacing: